test/Microsoft.ML.Tests/FeatureContributionTests.cs

```diff
@@ -1,344 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is the removal of this file intended?

Reply:
I have no idea why this is happening....

---
In reply to: [263192471](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263192471) [](ancestors = 263192471)